Makefile

```diff
@@ -9,7 +9,7 @@ docker_cmd  ?= docker
 docker_opts ?= --rm --tty --user "$$(id -u)"
 
 vale_cmd           ?= $(docker_cmd) run $(docker_opts) --volume "$${PWD}"/docs/modules/ROOT/pages:/pages vshn/vale:2.1.1 --minAlertLevel=error --config=/pages/.vale.ini /pages
-antora_preview_cmd ?= $(docker_cmd) run --rm --publish 2020:2020 --volume "${PWD}":/antora vshn/antora-preview:2.3.3 --style=syn --antora=docs
+antora_preview_cmd ?= $(docker_cmd) run --rm --publish 35729:35729 --publish 2020:2020 --volume "${PWD}":/preview/antora vshn/antora-preview:2.3.4 --style=syn --antora=docs
```

Comment:
Can we make those ports configurable? Extract them to variables on their own so the can be overwritten when calling make.

```
make docs-preview PREVIEW_HTTP_PORT=1234 PREVIEW_SOCKET_PORT=4321. Might come in handy if one has port conflicts.

Reply:
I'll keep this in mind for a future version!

Reply:
Just created an issue in the project for keeping track of that: https://github.com/vshn/antora-preview/issues/2